chroot-etc/croutonwheel.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
```

Comment:
Display is definitely available in croutonxinitrc-wrapper, so calling croutonwheel there should be fine.